.github/scripts/validate-run-checks.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const { repoToOwnerAndOwner, getPrCommits } = require('../../scripts/repo-utils');
+const { repoToOwnerAndOwner, getPrCommits } = require('./repo-utils');
```

Comment:
Why?

Reply:
We talk about it on Slack audio call.

Reply:
The action will run this file from the plugin directory, like all the other logics.